FRE/fre_make.json

```diff
@@ -191,7 +191,11 @@
                 },
                 "RUNenv": {
                     "description": "Commands needed at the beginning of a RUN in dockerfile",
-                    "type": ["array"],
+                    "anyOf": [
+                      {"type": "array"}, 
+                      {"type": "string"}
+                    ],
+                    "type": ["array", "string"],
```

Comment:
```
"type": ["array", "string"],
```
This line seems redundant, since the two `anyOf` cases require the type to be either a string or an array.

Reply:
Thanks for the insight.  It's much cleaner now